tools/get-alpine-pkg-source.sh

```diff
@@ -274,4 +277,4 @@ if [ -n "$urlfile" ]; then
 fi
 cd "$startdir" || exit
 # shellcheck disable=SC2002
-[ -z "$quiet" ] && echo "Collected $(du -sm "$outdir" | cut -f 1) Mbytes for $(cat "${OCPAIRS}" | wc -l) packages of source in $outdir"
+[ -z "$quiet" ] && echo "Collected $(du -sm "$outdir" | cut -f 1) Mbytes for $(cat "${OCPAIRS}" | wc -l) packages of source in $outdir"
```

Comment:
Unless my eyes are confused it seems like your change here is to remove the "return" (CR)  in the file. Is that correct? Useful?

Reply:
Still needs this.

Reply:
Fixed the script.